Makefile

```diff
@@ -1,4 +1,4 @@
-PROJECT="antenna"
+PROJECT="antenna-loadtests"
```

Comment:
this is ok for now.  we'll eventually replace this with our standardized Makefile and parameterize our project name into loadtest.env (or molotov.json).